<ui version="4.0" >
 <class>PkExample</class>
 <widget class="QMainWindow" name="PkExample" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PolicyKit-qt example</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Making the helper &lt;span style=" font-weight:600;">Play&lt;/span> requires the user to authenticate. The authorization is kept for the life time of the process&lt;/p>&lt;/body>&lt;/html></string>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="playPB" >
      <property name="text" >
       <string>Play!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Making the helper &lt;span style=" font-weight:600;">Cry&lt;/span> requires the user to authenticate. This is a one-shot authorization.&lt;/p>&lt;/body>&lt;/html></string>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cryPB" >
      <property name="text" >
       <string>Cry!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Making the helper &lt;span style=" font-weight:600;">Kick&lt;/span> requires a system administrator to authenticate. This instance overrides the defaults set in polkit_gnome_action_new().&lt;/p>&lt;/body>&lt;/html></string>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="kickPB" >
      <property name="text" >
       <string>Kick... (long)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4" >
      <property name="text" >
       <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Making the helper &lt;span style=" font-weight:600;">Delete&lt;/span> requires a system administrator to authenticate. Once authenticated, this privilege can be retained indefinitely.&lt;/p>&lt;/body>&lt;/html></string>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deletePB" >
      <property name="text" >
       <string>Delete!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5" >
      <property name="text" >
       <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Making the helper &lt;span style=" font-weight:600;">Bleed&lt;/span> requires the user to authenticate. Once authenticated, this privilege can be retained for the remainder of the desktop session.&lt;/p>&lt;/body>&lt;/html></string>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bleedPB" >
      <property name="text" >
       <string>Bleed!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6" >
      <property name="text" >
       <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">The &lt;span style=" font-weight:600;">Blow Up&lt;/span> action demonstrates the use of PolicyKit to drive a GtkToggleButton; it's an intuitive way to ask users to give up authorizations when they are done with them. E.g. the button is 'pressed in' exactly when the authorization is held. Toggling the button means obtaining resp. revoking the authorization in question.&lt;/p>&lt;/body>&lt;/html></string>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="listenPB" >
      <property name="text" >
       <string>Click to make changes...</string>
      </property>
      <property name="checkable" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7" >
      <property name="text" >
       <string>&lt;i>Tip: try editing /etc/PolicyKit/Policy.conf and see the proxy widgets update in real-time.&lt;/i>.</string>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions" >
    <property name="title" >
     <string>Actions</string>
    </property>
   </widget>
   <addaction name="menuActions" />
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="icons/icons.qrc" />
 </resources>
 <connections/>
</ui>
